src/components/SignUp/SignUp.scss

```diff
@@ -0,0 +1,39 @@
+@import '../../style/constants';
+
+label {
+  margin-top: 1rem !important;
+}
+
+.signup {
+  text-transform: uppercase;
+  margin-top: 6.75rem;
+}
+
+.confirmation__first_title {
+  font-family: Open Sans Bold;
```

Comment:
If all the application is using the same family maybe it's better to add this in a global css file and then override the necessary parts as the font weight

Reply:
I'm using `Open Sans Regular` as default

Reply:
Great, then you shouldn't have to detail again the font family, just adjust the weight with the `font-weight`